tests/conftest.py

```diff
@@ -0,0 +1,91 @@
+"""Configuration for the pytest test suite."""
+import warnings
+from pathlib import Path
+
+import morphio
+import neurom as nm
+import pytest
+
+
+def _load_morph_no_warning(filename):
+    all_warnings = [
+        j
+        for j in [getattr(morphio.Warning, i) for i in dir(morphio.Warning)]
+        if isinstance(j, morphio._morphio.Warning)
+    ]
+    morphio.set_ignored_warning(all_warnings, True)
+    morph = nm.load_morphology(filename)
+    morphio.set_ignored_warning(all_warnings, False)
+    return morph
+
+
+@pytest.fixture
+def DATA_PATH():
+    return Path(__file__).parent / "data"
+
+
+@pytest.fixture
+def H5_PATH(DATA_PATH):
+    return DATA_PATH / "h5" / "v1"
+
+
+@pytest.fixture
+def ASC_PATH(DATA_PATH):
+    return DATA_PATH / "neurolucida"
+
+
+@pytest.fixture
+def SWC_PATH(DATA_PATH):
+    return DATA_PATH / "swc"
+
+
+@pytest.fixture
+def SIMPLE_MORPHOLOGY(SWC_PATH):
+    return _load_morph_no_warning(SWC_PATH / "simple.swc")
+
+
+@pytest.fixture
+def SIMPLE_TRUNK_MORPHOLOGY(SWC_PATH):
+    return _load_morph_no_warning(SWC_PATH / "simple_trunk.swc")
+
+
+@pytest.fixture
+def SWC_MORPHOLOGY(SWC_PATH):
+    return _load_morph_no_warning(SWC_PATH / "Neuron.swc")
+
+
+@pytest.fixture
+def H5_MORPHOLOGY(H5_PATH):
+    return _load_morph_no_warning(H5_PATH / "Neuron.h5")
+
+
+@pytest.fixture
+def SWC_MORPHOLOGY_3PT(SWC_PATH):
+    return _load_morph_no_warning(SWC_PATH / 'soma' / 'three_pt_soma.swc')
+
+
+@pytest.fixture
+def MORPHOLOGY(SWC_PATH):
```

Comment:
Given that these objects make sense only in the context of specific tests, I am not fond of making them global fixtures..

Reply:
There are other tests that actually load the same files, they just don't use the fixture mechanism. I don't want to change all the tests to use fixtures for now, but we should prepare for it. But if you prefer we can move these fixtures to `tests/features` for now.